git_patrol_db.py

```diff
@@ -95,10 +96,11 @@ def __init__(self, asyncpg_pool):
     async with self.db_pool.acquire() as conn:
       insert_status = await conn.execute(
           '''INSERT INTO git_poll_journal (
-            git_poll_uuid, update_time, url, alias, refs)
-          VALUES ($1, $2, $3, $4, $5);
+            git_poll_uuid, update_time, url, alias, refs, ref_filters)
```

Comment:
outside the scope of this pull request, but if this ordering corresponds to your primary key, this results in lots of random writes.  The alternative would presumably be to make your update_time the outermost component of the key.

Reply:
Discussed offline. Thanks for the suggestion. I'll circle back to database layout once I get a better idea of the use cases at scale.